lib/models/settings_model.dart

```diff
@@ -8,23 +8,24 @@ import 'package:api_client/models/enums/default_timer_enum.dart';
 import 'package:api_client/models/model.dart';
 import 'package:api_client/models/enums/orientation_enum.dart';
 import 'package:api_client/models/weekday_color_model.dart';
+import 'enums/weekday_enum.dart';
```

Comment:
All current Dart files refer to the full path of the imported Dart file, i.e:
`import 'package:api_client/models/enums/weekday_enum.dart';`